File: aws-globalaccelerator-listener/src/main/java/software/amazon/globalaccelerator/listener/AcceleratorClientBuilder.java
package software.amazon.globalaccelerator.listener;

import com.amazonaws.services.globalaccelerator.AWSGlobalAccelerator;
import com.amazonaws.services.globalaccelerator.AWSGlobalAcceleratorClientBuilder;

public class AcceleratorClientBuilder {
    private static final String SUPPORTED_REGION = "us-west-2";
    static AWSGlobalAccelerator getClient() {
        return AWSGlobalAcceleratorClientBuilder.standard().withRegion(SUPPORTED_REGION).build();
    }
}

File: aws-globalaccelerator-listener/src/main/kotlin/software/amazon/globalaccelerator/listener/AcceleratorClientBuilder.kt
package software.amazon.globalaccelerator.listener

import com.amazonaws.services.globalaccelerator.AWSGlobalAccelerator
import com.amazonaws.services.globalaccelerator.AWSGlobalAcceleratorClientBuilder

/**
 * Singleton class for AWS Global Accelerator client
 */
object AcceleratorClientBuilder {
    private final const val SUPPORTED_REGION = "us-west-2"
    internal final val client: AWSGlobalAccelerator
        get() = AWSGlobalAcceleratorClientBuilder.standard().withRegion(SUPPORTED_REGION).build()
}
